packages/tomd/tests/test_emit.py

```
@@ -102,18 +102,16 @@ def test_front_matter_canonical_order():
-        "title: Canonical Test\n"
-        "document: P9999R0\n"
-        "date: 2026-04-28\n"
-        "intent: info\n"
-        "audience: LEWG\n"
-        "reply-to:\n"
-        '  - "Alice <a@x>"\n'
-        "---"
-    )
-    assert result == expected
+    lines = result.splitlines()
+    keys = [l.split(":")[0] for l in lines if l and not l.startswith((" ", "\t", "-")) and l != "---"]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)